Btms.Common/Json/UnknownTimeZoneDateTimeJsonConverter.cs

```
@@ -0,0 +1,42 @@
+using System.Globalization;
+using System.Text.Json;
+using System.Text.Json.Serialization;
+
+namespace Btms.Common.Json;
+
+[AttributeUsage(AttributeTargets.Property, AllowMultiple = false)]
+public class UnknownTimeZoneDateTimeJsonConverterAttribute(string propertyName) : JsonConverterAttribute
+{
+    public override JsonConverter? CreateConverter(Type typeToConvert)
+    {
+        return new UnknownTimeZoneDateTimeJsonConverter(propertyName);
+    }
+}
+
+public class UnknownTimeZoneDateTimeJsonConverter(string propertyName) : JsonConverter<DateTime>
+{
+    private const string JsonFormat = "yyyy-MM-ddTHH:mm:ss";
+
+    public override DateTime Read(ref Utf8JsonReader reader, Type typeToConvert, JsonSerializerOptions options)
+    {
+        if (typeToConvert != typeof(DateTime)) return DateTime.MinValue;
```

Can this ever happen? ie. `typeToConvert` not be a `DateTime`. As we might be better throwing instead of just returning `DateTime.MinValue`, otherwise the behaviour will go unnoticed and we'll start saving a different value.

---

Fair. Will throw for now